### DISCUSSION TITLE
Size of the organ load selection window

### DISCUSSION THREAD

**Discussion by willeke1234** [GrandOrgue/grandorgue#1051] [Help]
The window "Select organ to load" is very wide and the width can't be changed. How do I change the default size of this window? It used to be

![image](https://user-images.githubusercontent.com/97431382/155132642-f854d5c4-0368-4b2b-b16e-a13ecacaeea7.png)

and now is

![image](https://user-images.githubusercontent.com/97431382/155132668-15055e0e-ab19-49f3-bdec-91f3f7f9765b.png)

**Comment by paolo2504** [comment]
Size can only be enlarged vs right and down. Cannot be restricted horizontally.
In addition, sizes and position is not maintained after close (see post #1035).
